src/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+ 
+.card__image {
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+h1 {
+  margin: 0;
+}
+
+.card__title {
+  margin: 0 16px;
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 16px 16px;
```

Comment:
do not set top and bottom margins.
just one of them
instead horizontal margins set paddings for each side in .card 
![Screenshot from 2023-05-17 13-08-14](https://github.com/mate-academy/layout_product-cards/assets/33029864/456d5b9d-e4ef-4f3d-bad7-fb93c9dc5f53)